sov-enshrine-esdt-safe/src/deposit.rs

```diff
@@ -0,0 +1,82 @@
+use cross_chain::MAX_TRANSFERS_PER_TX;
+use multiversx_sc::imports::*;
+use operation::{aliases::OptionalValueTransferDataTuple, OperationData, TransferData};
+
+#[multiversx_sc::module]
+pub trait DepositModule:
+    multiversx_sc_modules::pause::PauseModule
+    + utils::UtilsModule
+    + cross_chain::deposit_common::DepositCommonModule
+    + cross_chain::execute_common::ExecuteCommonModule
+    + cross_chain::storage::CrossChainStorage
+    + cross_chain::events::EventsModule
+    + max_bridged_amount_module::MaxBridgedAmountModule
+{
+    #[payable("*")]
+    #[endpoint]
+    fn deposit(
```

Comment:
This function should be one to one with the function from `sov-esdt-safe`. Am I right? :thinking: 
If I am right, we should have a common deposit endpoint, but do this in different PR.